### ISSUE TITLE
New Chapter: Milwaukee, WI USA

### ISSUE THREAD

**Issue by afmsavage** [nodeschool/organizers#556]
Hello I would like to open a new chapter in the Milwaukee, WI area.  I have been learning JavaScript and Node now for about a month and really want to get in touch with other developers in the area.  I am a devops engineer for a smaller online education company and love it.  My Twitter is [@skumbagt](https://twitter.com/skumbagt) and here's a [link to my Dev.to account](https://dev.to/pantsme).  I would love to present and help organize events but don't know everything there is to know about JS/Node yet but I'm working on it!  Thanks!

**Comment by riyadhalnur** [comment]
Is this still valid? Sorry for the late response.

**Comment by afmsavage** [comment]
Yes I would like it to be.  I see there are a few other Javascript related Meetups my city but they aren't solely dedicated to NodeJS.  Might be nice to pair up with them and do something?

**Comment by riyadhalnur** [comment]
Hi @afmsavage , that's awesome. Do you have other co-organisers? We usually recommend at least 2 or 3 people to co-organise a given chapter.

**Comment by afmsavage** [comment]
Let me reach out to those other groups and see what their thoughts are. I will update you when I have a moment. If they are not interested, I have some coworkers who would be willing. We also host the Milwaukee Docker meetups and Wisconsin Agile meetups at my company's office so this could be another one that we are involved with company wise.

**Comment by Nobsjo86** [comment]
Interested
